test/dates/accessors.jl

```diff
@@ -118,7 +109,20 @@ function test_dates2(from,to)
         end
     end
 end
-test_dates2(0,2100)
+
+# test hour, minute, second
+let h=0, mi=0, s=0, ms=0, us=0, ns=0
+    for h = 0:23, mi = 0:59, s = 0:59,
+        ms in (0,1,500,999), us in (0,1,500,999), ns in (0,1,500,999)
```

Comment:
The segfault seems unrelated to this PR. I'm attempting a bisect to identify the root cause.